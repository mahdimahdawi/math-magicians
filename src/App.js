import UI from './components/Calculator';

/*eslint-disable */
const App = () => <UI />
 


export default App;
