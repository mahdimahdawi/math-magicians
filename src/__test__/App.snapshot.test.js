import React from 'react';
import App from '../App';
import renderer from 'react-test-renderer';

describe('Jest snapshot to test components', () => {
  test('Matches DOM Snapshot', () => {
    const domTree = renderer
      .create(
        <React.StrictMode>
          <App />
        </React.StrictMode>,
      )
      .toJSON();

    expect(domTree).toMatchSnapshot();
  });
});