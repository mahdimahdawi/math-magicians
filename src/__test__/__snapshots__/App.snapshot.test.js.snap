// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest snapshot to test components Matches DOM Snapshot 1`] = `
Array [
  <div
    className="header"
  >
    <div>
      <h1
        className="logo"
      >
        Math Magicians
      </h1>
    </div>
    <div
      className="menu"
    >
      <a
        aria-current="page"
        className="menu-link active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="menu-link"
      >
        |
      </span>
      <a
        className="menu-link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <span
        className="menu-link"
      >
        |
      </span>
      <a
        className="menu-link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </div>,
  <div
    className="home-page"
  >
    <h2>
      Welcome To Our Page !
    </h2>
    <p
      className="text"
    >
      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
    </p>
    <p
      className="text"
    >
      Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
    </p>
  </div>,
  <div
    className="footer"
  >
    <h3
      className="footer-text"
    >
      All Right Reserved Â©
    </h3>
  </div>,
]
`;
